pkg/api/v1/types.go

```diff
@@ -187,6 +187,12 @@ type ObjectMeta struct {
 	// from the list. If the deletionTimestamp of the object is non-nil, entries
 	// in this list can only be removed.
 	Finalizers []string `json:"finalizers,omitempty" patchStrategy:"merge" protobuf:"bytes,14,rep,name=finalizers"`
+
+	// Name of the cluster which the object belongs to. The federation apiserver/controllers use this to
```

Comment:
I think we can do without mentioning "federation" here.
Just say that this is used to distinguish resources with same name and namespace in different clusters.